sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/early_termination_policy.py

```diff
@@ -77,8 +77,8 @@ def __init__(
         *,
         delay_evaluation: int = 0,
         evaluation_interval: int = 0,
-        slack_amount: Optional[float] = 0,
-        slack_factor: Optional[float] = 0,
+        slack_amount: Union[float, None] = 0,
```

Comment:
For this one, `Optional[float]` is fine because it means the same thing as `Union[float, None]`. I think @annatisch's comment on the ApiView was to ask if the value really can be `None`. Please correct me if I'm wrong, @annatisch.

If you remove the `Optional` type and instead make these just `float`, does the mypy test on the PR fail?

Reply:
`slack_amount` and `slack_factor` should be able to be Optional:

![image](https://github.com/Azure/azure-sdk-for-python/assets/19650434/1ca9ecbc-01a3-4b48-93c9-2ab908f1d67c)

I will change it back to Optional[float]